### PR TITLE
Makes attention components more reusable

### PR DESCRIPTION
**Pull request by orelbn**
*Issue #:* /bcgov/entity#17039

*Description of changes:*
- Fixes issues with reusability for attention and folio components,
- Fixes state issue with attention component for MhrUnitNotes.
- Applies attention component to Mhr Filings.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1461-csvsnqxr.web.app

**Review by cameron-eyds — APPROVED** [review]
LGTM!

File: ppr-ui/src/components/common/Attention.vue [hunk 1]
```diff
@@ -17,12 +17,14 @@
       >
         <FormField
           :sectionId="sectionId"
-          :initialValue="getMhrAttentionReference"
+          :initialValue="initialValue"
           :inputTitle="config.inputTitle"
           :inputLabel="config.inputLabel"
+          :inputColWidth="hasWiderInput ? 10 : 9"
+          :labelColWidth="hasWiderInput ? 2 : 3"
```
Comment (dimak1) [inline ppr-ui/src/components/common/Attention.vue]: We have defaults for these props.

**Review by dimak1 — APPROVED** [review]
Looks good! Thanks for the updates.

File: ppr-ui/src/components/common/Attention.vue (hunk 1 above)
Comment (orelbn) [reply ppr-ui/src/components/common/Attention.vue]: Updated

**Review by doug-lovett — APPROVED** [review]
Looks great.

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1461-csvsnqxr.web.app

**Pull request merged**